workers/data_refinery_workers/processors/smasher.py

```diff
@@ -478,6 +483,102 @@ def sync_quant_files(output_path, files_sample_tuple, job_context: Dict):
         computed_file.get_synced_file_path(path=output_file_path)
     return num_samples
 
+def process_frame(inputs) -> Dict:
+    (job_context, computed_file, sample, index) = inputs
+    logger.debug('processing frame %s', index)
+    frame = {
+        "unsmashable_file": None,
+        "column_type": None,
+        "columns": None,
+        "data": None
+    }
+
+    def unsmashable(path):
+      frame['unsmashable_file'] = path
+      return frame
+
+    def smashable(data):
+      frame['data'] = data
+      return frame
+
+    try:
+        # Download the file to a job-specific location so it
+        # won't disappear while we're using it.
+
+        computed_file_path = job_context["work_dir"] + computed_file.filename
+        computed_file_path = computed_file.get_synced_file_path(path=computed_file_path)
+
+        # Bail appropriately if this isn't a real file.
+        if not computed_file_path or not os.path.exists(computed_file_path):
+            logger.error("Smasher received non-existent file path.",
+                computed_file_path=computed_file_path,
+                computed_file=computed_file,
+                dataset=job_context['dataset'],
```

Comment:
Can you change this to `job_context['dataset']`

Reply:
do you mean remove the comma?